core/controllers/base.py

```diff
@@ -565,9 +596,11 @@ def render_downloadable_file(self, file, filename, content_type):
         # We use this super in order to bypass the write method
         # in webapp2.Response, since webapp2.Response doesn't support writing
         # bytes.
-        super(webapp2.Response, self.response).write(file.getvalue())  # pylint: disable=bad-super-call
+        super(webapp2.Response, self.response).write(file.getvalue())  # type: ignore[misc] # pylint: disable=bad-super-call
```

Comment:
Mypy is unable to find here what super points to, and this is because super can accept `base class and self` but here we are passing different args.

Reply:
Comment updated!